egs/librispeech/ASR/conformer_ctc/pretrained.py

```diff
@@ -157,6 +167,13 @@ def get_parser():
         """,
     )
 
+    parser.add_argument(
+        "--lang-dir",
```

Comment:
If we provide `--lang-dir`, do we still need to pass `--words-file` and `--HLG` as they
are assumed to be inside `lang_dir`?